### PR TITLE
Add check_logfiles to ITL

### PR DESCRIPTION
**Pull request by KAMI911**
I am preparing a pull request to add check_logfiles plugin to ITL. The check_logfiles plugin is available from here:
https://labs.consol.de/nagios/check_logfiles/

### CODE REVIEW

File: itl/plugins-contrib.d/logfiles.conf [hunk 1]
```diff
@@ -0,0 +1,115 @@
+/*
+ * Icinga2 CheckCommand definitions to monitor log files using check_logfiles.
+ * The check_logfiles is a Plugin for Icinga which scans log files for specific patterns.
+ * Available from here: https://labs.consol.de/nagios/check_logfiles/
+ */
+
+object CheckCommand "log" {
```
Comment (dnsmichi) [inline itl/plugins-contrib.d/logfiles.conf]: Please use the full plugin name without the `check_` prefix. `log` is too generic, and also reserved for internal use.

File: itl/plugins-contrib.d/logfiles.conf [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	import "ipv4-or-ipv6"
+
+	command = [ PluginContribDir + "/check_logfiles.pl" ]
```
Comment (dnsmichi) [inline itl/plugins-contrib.d/logfiles.conf]: AFAIK the plugin is installed without the `.pl` suffix, configure generates this script like this.

File: doc/10-icinga-template-library.md [hunk 3]
```diff
@@ -2861,6 +2861,44 @@ ping_packets                     | **Optional.** The number of packets to send.
 ping_timeout                     | **Optional.** The plugin timeout in seconds. Defaults to 0 (no timeout).
 
 
+### Log Files <a id="plugins-contrib-log-files"></a>
```
Comment (dnsmichi) [inline doc/10-icinga-template-library.md]: Please move this section into the `Log Management` section, imho no need to duplicate this.

File: itl/plugins-contrib.d/logfiles.conf (hunk 1 above)
Comment (dnsmichi) [reply itl/plugins-contrib.d/logfiles.conf]: Besides, the documentation and parameters also refer to `logfiles` as check command name.

**Comment by mcktr** [comment]
I'd like to see this in the next release (if possible). I'll provide the needed changes for a a new review round.

**Comment by mcktr** [comment]
I addressed the needed changes with the last commit.

*  moved the documentation parts into the `Log Management` section
* moved the CheckCommand definition into `logmanagement.conf` and droppped `logfiles.conf`
* renamed the CheckCommand to `logfiles`
* removed the `.pl` suffix in the command attribute since the default installation will provide the plugin without the suffix
* removed the `-H` argument since its unknown to the plugin
```
./check_logfiles --H 10.2.66.4 --logfile /var/log/dpkg.log 
Unknown option: H
```
* fixed some typos in the documentation parts

**Comment by dnsmichi** [comment]
Thanks 👍

**Comment by ulyszk** [comment]
Thank you!

**Comment by KAMI911** [comment]
Sorry I was not able to provide the updated version of CheckCommand. It's good to see this was resolved.

**Pull request merged**